### PR TITLE
Issue #11584: WriteTag reports violation with confusing message when …

### PR DESCRIPTION
**Pull request by aayushRedHat**
Related: https://github.com/checkstyle/checkstyle/issues/11584 

Continuation of abandoned PR https://github.com/checkstyle/checkstyle/pull/12819

Diff regression config: https://gist.githubusercontent.com/strkkk/238b6c97492ff6ffdb3be1603deca8fc/raw/5db22461435a94c56a3f420be844cd07baf497f2/config_single_2.xml

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

File: src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/javadoc/WriteTagCheck.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
  Requires user defined Javadoc tag to be present in Javadoc comment with defined format.
  To define the format for a tag, set property tagFormat to a regular expression.
  Property tagSeverity is used for severity of events when the tag exists.
+ No violation reported in case there is no javadoc.
```
Comment (romani) [inline src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/javadoc/WriteTagCheck.xml:11]: Please add to examples some method without javadoc, to make it very clear that there is no validation happening.

@aayushRedHat , it looks like this update is kind of breaking changes in behavior. Do you see what Checks should be used in addition to let users get behavior how it was before?

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Comment by nrmancuso** [comment]
@aayushRedHat do you plan to continue to work on this PR?

**Comment by romani** [comment]
@MohanadKh03, please help us to finish this PR.
Reuse code , create new PR and I hope we can have enough energy to merge your PR.

**Comment by romani** [comment]
Will be done at https://github.com/checkstyle/checkstyle/pull/15844

**Pull request closed without merging**